src/Microsoft.Azure.ServiceBus/Extensions/MessageInterOpExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Extensions
```

Comment:
>  we might be able to put them under the same extensions namespace

"What if" case? 🙂
Reason I've suggested this change to the namespace is because would rather see partitioning by major role (Interop, connectivity, etc) rather than grouping by implementation (extension methods, singletons, etc).